templates/easy_admin/talk/authors.html.twig

```diff
@@ -0,0 +1,7 @@
+{% for submit in item.submits %}
+    <ul style="margin-bottom:0">
```

Comment:
I would put the `ul` before the first for, or you'll get many ul in this table cell.
And IMHO, we will have lot's of duplicate content here.
If a user submit 5 talks. He'll appear here 5 times.
You should unique the list

Reply:
Even if most of time there will be 1 speaker, if a talk is submitted 2 times with 2 different speakers couple (let say Alice and Bob, and Alice and Jack), we will see "Alice, Bob, Jack", which is not very relevant, don't you think? 
May be unique the list, but "by submit"
1st submit: Alice
2nd submit: Alice
3rd submit: Alice and Bob
4th submit: Alice and Jack
5th submit: Alice and Bob
will result to:
Alice, Alice + Bob, Alice + Jack

WDYT?